addons/upgrade/to-13.1-move-ntlm-auth-to-rest.pl

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+addons/upgrade/to-10.1-move-radius-configuration-parameter.pl
+
+=cut
+
+=head1 DESCRIPTION
+
+Move radius configuration parameters to associated new files
+
+=cut
+
+use strict;
+use warnings;
+use lib qw(/usr/local/pf/lib /usr/local/pf/lib_perl/lib/perl5);
+use pf::IniFiles;
+use pf::file_paths qw($authentication_config_file $domain_config_file);
+use pf::util;
+use Digest::MD4;
+use Encode;
+use MIME::Base64;
+
+my $ini = pf::IniFiles->new(-file => $domain_config_file, -allowempty => 1);
+
+unless ($ini) {
+    exit;
+}
+
+my $updated = 0;
+my $ntlm_auth_host = "127.0.0.1";
+my $ntlm_auth_port = 4999;
+
+for my $section (grep {/^\S+$/} $ini->Sections()) {
+    print("Generating config for section: $section\n");
+    $ntlm_auth_port += 1;
+
+    if ($ini->exists($section, 'machine_account_password')) {
+        print("  Section: ", $section, " already has machine_account and machine_account_password set. skipped\n");
```

Comment:
Use consistent wording in all (3) locations:
* . skipped\n
* , section $section skipped\n
* , Skipping section $section\n